letsencrypt/tests/client_test.py

```diff
@@ -100,22 +101,35 @@ def _mock_obtain_certificate(self):
         self.acme.request_issuance.return_value = mock.sentinel.certr
         self.acme.fetch_chain.return_value = mock.sentinel.chain
 
-    def _check_obtain_certificate(self):
-        self.client.auth_handler.get_authorizations.assert_called_once_with(
-            ["example.com", "www.example.com"])
+    def _check_obtain_certificate(self, skip_authz=False):
+        if skip_authz:
+            self.client.auth_handler.get_authorizations.assert_not_called()
```

Comment:
`assert_not_called` requires `mock>=1.3.0`, so it's a NOP e.g. in Python 2.6. Please fix